src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -2215,6 +2215,11 @@ protected internal virtual CSharpSyntaxNode GetBindableSyntaxNode(CSharpSyntaxNo
                         }
                         break;
 
+                    case SyntaxKind.ComplexElementInitializerExpression:
+                    case SyntaxKind.CollectionInitializerExpression:
+                    case SyntaxKind.ObjectInitializerExpression:
```

Comment:
Do we ever create bound nodes for this syntax? If not, then it feels inappropriate to handle them specially here because it doesn't matter what node we try to bind, we do not get a bound node for them.